model/src/main/java/uk/gov/hmcts/payment/api/model/PaymentStatus.java

```diff
@@ -16,6 +16,8 @@
 @Table(name = "payment_status")
 public class PaymentStatus {
 
+    public final static PaymentStatus CREATED = new PaymentStatus("created", "created");
```

Comment:
static instance..? might misuse later..

Reply:
Avoids one extra query to the database to get the object for the query.